### PR TITLE
Remove early access for CVC Recollection

### PR DESCRIPTION
**Pull request by wooj-stripe**
## Summary
Removing early access @_spi

## Motivation
GA

## Testing
Relying on existing tests

## Changelog
[Added] CVC Recollection is now in GA

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## X.Y.Z X-Y-Z
 ### PaymentSheet
 * [Added] The ability to customize the disabled colors of the primary button with `PaymentSheetAppearance.primaryButton.disabledBackgroundColor` and `PaymentSheetAppearance.primaryButton.disabledTextColor`.
+* [Added] CVC Recolleciton is now in GA
```
Comment (porter-stripe) [inline CHANGELOG.md:4]: Do we have any links to docs we could add?

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift [hunk 2]
```diff
@@ -34,27 +34,6 @@ public extension PaymentSheet {
...
         ) -> Void
 
-        /// Creates a `PaymentSheet.IntentConfiguration` with the given values
-        /// - Parameters:
-        ///   - mode: The mode of this intent, either payment or setup
-        ///   - paymentMethodTypes: The payment method types for the intent
-        ///   - onBehalfOf: The account (if any) for which the funds of the intent are intended
-        ///   - paymentMethodConfigurationId: Configuration ID (if any) for the selected payment method configuration
-        ///   - confirmHandler: A handler called with payment details when the user taps the primary button (e.g. the "Pay" or "Continue" button).
-        public init(mode: Mode,
```
Comment (porter-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift:44]: Might be missing something, but how is this not a breaking change?

Comment (porter-stripe) [reply StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift:44]: I guess the new init with `requireCVCRecollection` which has a default value takes it place?

Comment (wooj-stripe) [reply StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift:44]: Yea, the diff is weird looking. It should not be a breaking change.

**Review by porter-stripe — DISMISSED** [review]

**Review by porter-stripe — APPROVED** [review]

**Pull request merged**